apps/darts-modernisation/base/kustomize.yaml

```diff
@@ -9,5 +9,5 @@ spec:
     substitute:
       NAMESPACE: "darts-modernisation"
       TEAM_NOTIFICATION_CHANNEL: "darts-builds"
-      WI_NAME: darts
+      WI_NAME: darts-prod-mi
```

Comment:
I think this should be left as is

Reply:
yeah I think the scripts overwrote it, will push an update now